components/project-loader/project-loader.jsx

```diff
@@ -117,37 +123,30 @@ export default React.createClass({
   },
   renderLearnMoreNotice() {
     if(!this.props.featured) return null;
-  
-    return <div><p>Discover & collaborate on projects for a healthy internet. <a href="https://www.mozillapulse.org/entry/120">Learn more</a>.</p></div>;
-  },
-  renderSearchResult() {
-    if (!this.props.search || this.state.loadingData) return null;
 
-    let total = this.state.totalMatched,
-      plural = (total === 0 || total > 1), // because "0 results"
-      term = this.props.search,
-      searchResultNotice = `${total} result${plural ? `s` : ``} found for ‘${term}’`;
-
-    return <p>{searchResultNotice}</p>;
+    return <div><p>Discover & collaborate on projects for a healthy internet. <a href="https://www.mozillapulse.org/entry/120">Learn more</a>.</p></div>;
   },
   renderEntryCounter() {
     if (this.state.loadingData) return null;
-    if (!this.props.issue && !this.props.tag) return null;
-    
-    return <p>{this.state.totalMatched} result{this.state.totalMatched > 0 ? `s` : ``} found</p>;
+    if (!this.props.search && !this.props.issue && !this.props.tag) return null;
+
+    let counterText = `${this.state.totalMatched} result${this.state.totalMatched > 0 ? `s` : ``} found`;
```

Comment:
This needs to be `this.state.totalMatched === 1? '' : 's'`:

- 0 result**s** found
- 1 result found
- 2 result**s** found.